modules/apps/foundation/frontend-js/frontend-js-web/build.gradle

```diff
@@ -30,12 +30,6 @@ buildClay {
 
 	eachFile new StripPathSegmentsAction(3)
 
```

Comment:
@jbalsas, can you remove this empty line?

Reply:
Of course!